main/py-filelock/APKBUILD

```
@@ -12,8 +12,8 @@ depends=""
 checkdepends="pytest"
 makedepends="python2-dev python3-dev py-setuptools"
 subpackages="py3-$_pkgname:_py3 py2-$_pkgname:_py2"
-source="$pkgname-$pkgver.tar.gz::https://github.com/benediktschmitt/$pkgname/archive/v$pkgver.tar.gz"
-builddir="$srcdir"/$pkgname-$pkgver
+source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz"
```

```suggestion
source="https://files.pythonhosted.org/packages/source/f/$_pkgname/$_pkgname-$pkgver.tar.gz"
```

This was a bashism that we don't use.

---

Cool, I applied the change and rebased. I'll use it in future PRs